src/i965_drv_video.c

```diff
@@ -618,7 +618,8 @@ i965_QueryConfigEntrypoints(VADriverContextP ctx,
        if (HAS_H264_MVC_DECODING_PROFILE(i965, profile))
             entrypoint_list[n++] = VAEntrypointVLD;
 
-       if (HAS_H264_MVC_ENCODING(i965))
+       if (HAS_H264_MVC_ENCODING(i965) &&
```

Comment:
This check looks too hack.
The MVC encoding can be disabled in skl/kbl_hw_codec_info.